ikasaneip/platform/ikasan-setup/src/main/resources/providers-properties.xml

```diff
@@ -1296,9 +1293,7 @@
 					INSERT
 					INTO PolicyLinkType ( Name, TableName )
 					VALUES ( 'Mapping Configuration',  'MCSMappingConfiguration' )
-					
-					INSERT INTO AuthenticationMethod (Id, Method)
-					VALUES (1, 'AUTH_METHOD_LOCAL')
+				
```

Comment:
What is the reason for removing from init script? Will there be local ldap to authenticated against?

INSERT INTO AuthenticationMethod (Id, Method)		
-					VALUES (1, 'AUTH_METHOD_LOCAL')